.dockerignore

```diff
@@ -2,4 +2,4 @@
 **/*.pyc
 **/.cache/
 **/compilation_cache/
-src/macros/externs.sem
```

Comment:
Ah, totally forgot to remove this when I removed the `generateExterns` call.  My bad.